frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -327,25 +305,14 @@ function InsightCardInternal(
                     rename={rename}
                     duplicate={duplicate}
                     moveToDashboard={moveToDashboard}
-                    setPrimaryHeight={setMetaPrimaryHeight}
                     areDetailsShown={areDetailsShown}
                     setAreDetailsShown={setAreDetailsShown}
                     showEditingControls={showEditingControls}
                     showDetailsControls={showDetailsControls}
                     moreButtons={moreButtons}
                 />
                 {insight.query ? (
-                    <div
```

Comment:
Same here - we din't care about heights as the child is always fixing to the parent